discretisedfield/field.py

```diff
@@ -2021,25 +2020,17 @@ def diff(self, direction, order=1):
                     padded_array, self.mesh.cell[direction_idx], axis=direction_idx
```

Comment:
Regarding the comment two lines above: Isn't this a case where we can only compute forward/backward differences?